src/fhirtypes/ElementDefinition.ts

```diff
@@ -633,9 +631,15 @@ export class ElementDefinition {
       this.checkSumOfSliceMins(max);
       // Check that new max >= every individual child max
       const slices = this.getSlices();
-      const overMaxChild = slices.find(child => child.max === '*' || parseInt(child.max) > maxInt);
-      if (!isUnbounded && overMaxChild) {
-        throw new InvalidMaxOfSliceError(overMaxChild.max, overMaxChild.sliceName, max);
+      const overMaxChildren = slices.filter(
+        child => child.max === '*' || parseInt(child.max) > maxInt
+      );
+      if (!isUnbounded && overMaxChildren.length > 0) {
+        logger.warn(
+          `Max of element ${this.id} is > max of slice${
+            overMaxChildren.length > 1 ? 's' : ''
```

Comment:
I think that a colon here would help it read better, particularly in the case of multiple slices.
```suggestion
            overMaxChildren.length > 1 ? 's:' : ':'
```